eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -89,8 +89,10 @@ else {
     # setting variable to reference the pull request by number
     Write-Host "##vso[task.setvariable variable=Submitted.PullRequest.Number]$($resp.number)"
 
-    Add-GitHubPullRequestReviewers -RepoOwner $RepoOwner -RepoName $RepoName -PrNumber $resp.number `
-    -Users $UserReviewers -Teams $TeamReviewers -AuthToken $AuthToken
+    if ($UserReviewers -or $TeamReviewers) {
```

Comment:
Why not do this in the Add-GitHubPullRequestReviewers function?

Reply:
Mitch pointed out that you can fire post request even when the body is empty. And I though you requested I move the check to where it is the function is called?

Reply:
I was expecting the check to be wherever we create the body and then we could check that the body is empty or not not make the rest call in that case. I believe that place is likely the Add-GitHubPullRequestReviewers function.

Reply:
To be fair I'm also fine with this check here as well but I do think we could also do a similar check in Add-GitHubPullRequestReviewers.